internal/gps/cmd.go

```diff
@@ -196,12 +198,31 @@ func runFromRepoDir(ctx context.Context, repo vcs.Repo, timeout time.Duration, c
 	return c.combinedOutput(ctx)
 }
 
+func getenvDuration(key string, def time.Duration) time.Duration {
+	seconds, err := strconv.Atoi(os.Getenv(key))
+
+	if err != nil {
+		return def
```

Comment:
This gives no indication to the user when their environment variable is set but isn't convertible.